projects/demo/src/modules/services/breakpoint/examples/1/template.html

```diff
@@ -0,0 +1,24 @@
+<p>Change the viewport of this window to see changes in breakpoint</p>
+
+<table class="tui-space_top-4">
+    <thead>
+        <tr>
+            <th>CSS</th>
+            <th>Breakpoint service</th>
+        </tr>
+    </thead>
+    <tbody>
+        <td>
+            <div class="mobile">Mobile breakpoint</div>
+            <div class="desktop-small">Desktop small breakpoint</div>
+            <div class="desktop-large">Desktop large breakpoint</div>
+        </td>
+        <td>
+            <ng-container *ngIf="breakpoint$ | async as breakpoint">
+                <div *ngIf="breakpoint === 'mobile'">Mobile breakpoint</div>
```

Comment:
Should we deprecate this token ?

https://github.com/Tinkoff/taiga-ui/blob/872919e27828ece252eeb5c85b3c99371bbfb82b/projects/core/tokens/is-mobile-resolution.ts#L10-L13

Reply:
I mean to add js-code comment:

```ts
/** 
  * @deprecated use {@link services/breakpoint-service TuiBreakpointService}
  * TODO: drop in v4.0
  * Mobile resolution stream for private providers 
  */ 
 export const TUI_IS_MOBILE_RES = new InjectionToken<Observable<boolean>>( 
```

Reply:
✅